README.md

```diff
@@ -174,6 +174,12 @@ specify a group using the `--group` option.
 You can specify a version using the `--version` option. This version string is
 used when publishing SDKs to registries (e.g. npm).
 
+#### Running locally
+
+By default, the generation runs in the cloud. If you want to run it on your local machine, you can use the `--local` option. _This will run the compiler in a Docker container._
```

Comment:
```suggestion
By default, the generation runs in the cloud. If you want to run it on your local machine, you can use the `--local` option. This will run each generator in a Docker container.
```